src/qnorm.rs

```diff
@@ -0,0 +1,159 @@
+use crate::dpq::*;
+use crate::libc::*;
+use crate::nmath::*;
+use crate::rmath::*;
+
+/// Compute the quantile function for the normal distribution.
+///
+/// The algorithm AS 241 of Wichura is used,
+///     and has been improved for the very extreme tail (and log_p=TRUE)
+///
+/// REFERENCE
```

Comment:
```suggestion
/// ## REFERENCE
```

Let's try to keep like this?
I know that C docstrings are awful, but we could show how to do better?